apps/backend/src/prof-ratings/prof-ratings.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { Body, Controller, Delete, Get, Param, ParseIntPipe, Patch, Post } from '@nestjs/common';
+
+import { CreateProfRatingDto } from './dto/create-prof-rating.dto';
+import { UpdateProfRatingDto } from './dto/update-prof-rating.dto';
+import { ProfRating } from './entities/prof-rating.entity';
+import { ProfRatingsService } from './prof-ratings.service';
+
+@Controller('prof-ratings')
+export class ProfRatingsController {
+  constructor(private readonly profRatingsService: ProfRatingsService) {}
+
+  @Post()
+  create(@Body() createProfRatingDto: CreateProfRatingDto): Promise<ProfRating> {
+    return this.profRatingsService.create(createProfRatingDto);
+  }
+
+  @Get()
+  findAll(): Promise<ProfRating[]> {
+    return this.profRatingsService.findAll();
+  }
+
+  @Get(':id')
+  findOne(@Param('id', ParseIntPipe) id: number): Promise<ProfRating | null> {
```

Comment:
Here you can just return Promise<ProfRating>. If it is not found, we throw an exception anyway. This is true for all Promises here.